test/hotspot/jtreg/vmTestbase/vm/runtime/defmeth/ConflictingDefaultsTest.java

```diff
@@ -82,6 +82,8 @@ public static void main(String[] args) {
      * class C implements I, J {}
      *
      * TEST: C c = new C(); c.m() ==> ICCE
+     * TEST: I c = new C(); c.m() ==> ICCE
+     * TEST: J c = new C(); c.m() ==> ICCE
```

Comment:
So this is the generator code that generates the defmeth tests, but iirc, the tests are regenerated and the generated tests were the ones checked in and run.

And shouldn't the test from commit 87e30fd80172c5bd6748f140ddf6cd19adb62764 now fail?

Reply:
I'm not aware of any checked in generated tests. I added the new tests here and they run.

Why should the test for loader-constraints fail? I fixed that issue just in a different way to the original fix. Hence the test passes as it should.

Reply:
Got it now.

Thanks for verifying that the new tests run.